### ISSUE TITLE
Error report. Check for me. Thank you

### ISSUE THREAD

**Issue by ghost** [tkn963/wrath-of-the-vanilla#2]
root@VM-0-9-ubuntu:/usr/local/azerothcore/build# make 2>&1 | grep 'error'
/usr/local/azerothcore/modules/mod-wrath-of-the-vanilla/src/BlackrockSpire/instance_blackrock_spire.cpp:359:54: fatal error: use of undeclared identifier 'EVENT_DARGONSPIRE_ROOM_STORE'; did you mean 'EVENT_DRAGONSPIRE_ROOM_STORE'?
1 error generated.

**Comment by noisiver** [comment]
Yes, this code is out-of-date. I'm not maintaining it since switching to a different implementation of progression. I decided to leave this here if someone wanted to grab something from it.

I'm going to make the repo as archive now though.
